utils.go

```diff
@@ -58,13 +61,14 @@ func CapellaRequestToProtoRequest(block *capella.SubmitBlockRequest) *SubmitBloc
 			GasUsed:       block.ExecutionPayload.GasUsed,
 			Transactions:  transactions,
 			Withdrawals:   withdrawals,
+			BlobGasUsed:   block.ExecutionPayload.BlobGasUsed,
+			ExcessBlobGas: block.ExecutionPayload.ExcessBlobGas,
```

Comment:
this is good but we'll also need the actual blob data that's in `*deneb.SubmitBlockRequest`
```
type BlobsBundle struct {
    Commitments []deneb.KZGCommitment `ssz-max:"4096" ssz-size:"?,48"`
    Proofs      []deneb.KZGProof      `ssz-max:"4096" ssz-size:"?,48"`
    Blobs       []deneb.Blob          `ssz-max:"4096" ssz-size:"?,131072"`
}
```

Reply:
Done